lib/cylc/cfgspec/gcylc.py

```diff
@@ -40,6 +40,8 @@
                              options=["small","medium","large", "extra large"]),
     'sort by definition order' : vdr( vtype='boolean', default=True ), 
     'task filter highlight color' : vdr(vtype='string', default='PowderBlue'),
+    'initial orientation' : vdr(vtype='string', default="vertical",
```

Comment:
I'd be slightly tempted to use something like a boolean titled horizontal view or somesuch and just reference that later rather than doing `initial orientation == side-by-side`. @benfitzpatrick - thoughts?